testing/jenkins_remote_access.py

```diff
@@ -87,6 +89,24 @@ def mesosSlaveInfo = new MesosSlaveInfo(
 Jenkins.instance.items.each { job -> job.delete() }
 """
 
+JENKINS_JOB_FAILURES = """
+activeJobs = hudson.model.Hudson.instance.items.findAll{job -> !(job instanceof Folder) && job.isBuildable()}
+println("successjobs = " +activeJobs.size())
+failedRuns = activeJobs.findAll{job -> job.lastBuild != null && !(job.lastBuild.isBuilding()) && job.lastBuild.result == hudson.model.Result.FAILURE}
```

Comment:
It is possible for a job to also have a result of `ABORTED` (and maybe canceled?). Do we want to count those? Is the intention to find failed jobs or jobs that were not successful?